src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,18 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (std::string title)
+{
+    if (auto j = debugJournal().debug())
+    {
+        j << title;
+        for (auto const& frame : beast::getStackBacktrace ())
+            j << "    " << frame;
+        j << "----";
+    }
+}
```

Comment:
Thanks @HowardHinnant.  Is that because Apple implemented a much better equivalent to gcc's `execute_native_thread_routine(void* __p)`?  https://chromium.googlesource.com/chromiumos/third_party/gcc/+/5cfcf8452e8f755d2b2899d96f93afdb4672de16/gcc/gcc-4.6.0/libstdc++-v3/src/c++11/thread.cc